modules/afamqp/afamqp-parser.c

```diff
@@ -42,6 +42,8 @@ static CfgLexerKeyword afamqp_keywords[] =
   { "persistent",   KW_PERSISTENT },
   { "username",     KW_USERNAME },
   { "password",     KW_PASSWORD },
+  { "max_channel",     KW_MAX_CHANNEL },
```

Comment:
just a nitpick : I know that this indentation "style" is part of the original implementation, so nothing to do here from your part, but when I saw the indentation patch.... well I just hoped that this ugly formatting has been also covered by that patch :)

Reply:
Yeah, I should have. But those refactor/indent style change I just cherry picked from my other work in progress afamqp branch, because I do not not if that other branch is going to become a PR or not.